### PR TITLE
Navigation position style fixes [storage]

### LINKED ISSUE #2087: [Storage] Element positioning issue on slide-out navigation (Android)
**Description**
When you navigate back and forth between the buckets page and cids page the positioning of the navigation elements will break. 

**I could only reproduce this on Android, in either Chrome or the Metamask app browser** 

**Steps to Reproduce**
1. Log into storage on Android
2. Navigate back and forth between bucket page and cid page

**Observed behavior**

https://user-images.githubusercontent.com/3982665/162813244-b4350995-e5b2-4a55-b3b0-e886624c8066.mp4

**Expected behavior**
Menu elements should not change position. 

**Desktop:**
 - OS: Android 12
 Browser: Chrome / Android Metamask app
 - Version : 100.0.4896.79

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #2087

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

### CODE REVIEW

**Review by juans-chainsafe — CHANGES_REQUESTED** [review]
Looking good! but I have a problem, only in mobile of course, using Chrome or Metamask app:
In CIDs table I only see the columns "Name", "Cid" and the 3 dots. I can't move left or right to see other columns ("Created", "Size", "Status")
IMG_BB12982AE042-1

**Comment by tanmoyAtb** [comment]
> Looking good! but I have a problem, only in mobile of course, using Chrome or Metamask app: In CIDs table I only see the columns "Name", "Cid" and the 3 dots. I can't move left or right to see other columns ("Created", "Size", "Status") IMG_BB12982AE042-1

This was very intentional actually. If you have multiple columns in the table, the alignment of the whole page breaks and looks odd. 
I wanted to add a quick fix to keep the navigation usable and I don't think anyone would look for CID information on mobile, therefore omitted the columns.

**Review by juans-chainsafe — APPROVED** [review]
@tanmoyAtb well, yes, make sense to only have those columns in mobile, but just in case I want @asnaith opinion on that to be sure!

👍 I agree with Tanmoy's rationale, it's not worth the time investment in making the view support all columns as are expecting very low usage of Storage UI on mobile devices.

**Comment by asnaith** [comment]
> @tanmoyAtb well, yes, make sense to only have those columns in mobile, but just in case I want @asnaith opinion on that to be sure!
> 
> 

👍 I agree with Tanmoy's rationale, it's not worth the time investment in making the view support all columns as are expecting very low usage of Storage UI on mobile devices.

File: packages/storage-ui/src/Components/Pages/CidsPage.tsx [hunk 1]
```diff
@@ -220,34 +232,42 @@ const CidsPage = () => {
...
+                  data-cy="table-header-status"
+                  sortButtons={false}
+                  align="center"
+                >
+                  <Trans>Status</Trans>
+                </TableHeadCell>
+              }
+              {desktop &&
+                <TableHeadCell>{/* IPFS Gateway */}</TableHeadCell>
+              }
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Pages/CidsPage.tsx]: We can group this all can't we?
```suggestion
              {desktop && <>
                <TableHeadCell
                  data-cy="table-header-created"
                  sortButtons={true}
                  onSortChange={() => handleSortToggle("date_uploaded")}
                  sortDirection={sortColumn === "date_uploaded" ? sortDirection : undefined}
                  sortActive={sortColumn === "date_uploaded"}
                  align="center"
                >
                  <Trans>Created</Trans>
                </TableHeadCell>
                <TableHeadCell
                  data-cy="table-header-size"
                  sortButtons={true}
                  onSortChange={() => handleSortToggle("size")}
                  sortDirection={sortColumn === "size" ? sortDirection : undefined}
                  sortActive={sortColumn === "size"}
                  align="center"
                >
                  <Trans>Size</Trans>
                </TableHeadCell>
                <TableHeadCell
                  data-cy="table-header-status"
                  sortButtons={false}
                  align="center"
                >
                  <Trans>Status</Trans>
                </TableHeadCell>
                <TableHeadCell>{/* IPFS Gateway */}</TableHeadCell>
              </>}
```

File: packages/storage-ui/src/Components/Elements/CidRow.tsx [hunk 2]
```diff
@@ -80,35 +81,43 @@ const CidRow = ({ pinStatus }: Props) => {
...
+      {desktop &&
+        <TableCell className={classes.externalIconCell}>
+          {pinStatus.status === "pinned" && (
+            <ExternalIcon
+              className={classes.icon}
+              onClick={() => window.open(`${trimChar(IPFS_GATEWAY, "/")}/${pinStatus.pin?.cid}`)}
+            />
+          )}
+        </TableCell>
+      }
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Elements/CidRow.tsx]:
```suggestion
      {desktop && <>
        <TableCell>
          {dayjs(pinStatus.created).format("DD MMM YYYY h:mm a")}
        </TableCell>
        <TableCell>
          {pinStatus.info?.size ? formatBytes(pinStatus.info?.size, 2) : "-"}
        </TableCell>
        <TableCell>
          {pinStatus.status}
        </TableCell>
        <TableCell className={classes.externalIconCell}>
          {pinStatus.status === "pinned" && (
            <ExternalIcon
              className={classes.icon}
              onClick={() => window.open(`${trimChar(IPFS_GATEWAY, "/")}/${pinStatus.pin?.cid}`)}
            />
          )}
        </TableCell>
      </>}
```

**Review by Tbaut — COMMENTED** [review]
Just a nit, the code is looking good otherwise

**Comment by tanmoyAtb** [comment]
> Just a nit, the code is looking good otherwise

Updates are in.

**Review by Tbaut — APPROVED** [review]
Code looks good

**Pull request merged**